docs/_docs/examples/deid-dataset.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Deidentify a Pydicom Dataset
```

Comment:
I'm really excited to add this!  Note that we already have up to "5" in the examples folder, so either you can adjust the others order (and keep 3 here) or this should be 6. https://github.com/pydicom/deid/tree/master/docs/_docs/examples